ledger-service/http-json/src/main/scala/com/digitalasset/http/LedgerClientJwt.scala

```diff
@@ -115,29 +132,62 @@ object LedgerClientJwt {
 
   private def bearer(jwt: Jwt): Some[String] = Some(jwt.value: String)
 
+  private def logLedgeClientCallTime[T, C](
+      clientCallName: String,
+      shouldLog: Boolean,
```

Comment:
Maybe the loglevel could be added to the `LoggingContextOf`. In a more direct usage of slf4j you would just use `logger.isDebugEnabled`

Reply:
I am thinking to make this `isEnable` def public in order to get access to the `isDebugEnabled()`
https://github.com/digital-asset/daml/blob/main/libs-scala/contextualized-logging/src/main/scala/com/digitalasset/logging/LeveledLogger.scala#L89

Not sure if it is a good idea. I have pushed the change. Please have a look.